pkg/app/piped/livestatestore/ecs/store.go

```diff
@@ -16,36 +16,133 @@ package ecs
 
 import (
 	"context"
+	"fmt"
+	"sync/atomic"
+	"time"
 
+	"github.com/aws/aws-sdk-go-v2/service/ecs/types"
+	"github.com/pipe-cd/pipecd/pkg/model"
```

Comment:
nits.

This `github.com/pipe-cd/pipecd/pkg/model` import should be in the last group.